 PackageInstaller: Fix FC when tapping on a downloaded apk notification 2

: FATAL EXCEPTION: AsyncTask #1
: Process: com.android.packageinstaller, PID: 11820
: java.lang.RuntimeException: An error occurred while executing doInBackground()
: at android.os.AsyncTask.done(AsyncTask.java:318)
: at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
: at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
: at java.util.concurrent.FutureTask.run(FutureTask.java:242)
: at android.os.AsyncTask.run(AsyncTask.java:243)
: at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
: at java.util.concurrent.ThreadPoolExecutor.run(ThreadPoolExecutor.java:607)
: at java.lang.Thread.run(Thread.java:761)
: Caused by: java.lang.SecurityException: Permission Denial: 
 reading com.android.providers.downloads.DownloadProvider 
 uri content://downloads/all_downloads/101 from pid=11820, 
 uid=10025 requires android.permission.ACCESS_ALL_DOWNLOADS, or grantUriPermission()

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 39e9f3c..338c16a 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -21,6 +21,7 @@
     <uses-permission android:name="android.permission.MANAGE_APP_OPS_RESTRICTIONS" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.HIDE_NON_SYSTEM_OVERLAY_WINDOWS" />
+    <uses-permission android:name="android.permission.ACCESS_ALL_DOWNLOADS" />
 
     <uses-permission android:name="com.google.android.permission.INSTALL_WEARABLE_PACKAGES" />
 
